package.json

```diff
@@ -17,18 +17,21 @@
     "axios": "^0.21.1",
```

Comment:
Please revert the changes to this file, i do not think you need to change the package.json and also the package-lock.json with your changes :)